packages_rs/nextclade/src/tree/tree_attach_new_nodes.rs

```diff
@@ -107,7 +107,7 @@ fn add_child(node: &mut AuspiceTreeNode, result: &NextcladeOutputs) {
         region: Some(TreeNodeAttr::new(AUSPICE_UNKNOWN_VALUE)),
         country: Some(TreeNodeAttr::new(AUSPICE_UNKNOWN_VALUE)),
         division: Some(TreeNodeAttr::new(AUSPICE_UNKNOWN_VALUE)),
-        placement_prior: Some(TreeNodeAttr::new("")),
+        placement_prior: None,
         alignment: Some(TreeNodeAttr::new(&alignment)),
         missing: Some(TreeNodeAttr::new(&format_missings(&result.missing, ", "))),
```

Comment:
None means it won't appear in the tree at all - not as NAN, right? Float defines nan, hence the question.

Reply:
The full type is

https://github.com/nextstrain/nextclade/blob/650a5eb4a9811d4c8b4bbb9020d7a6c0a5c4a92c/packages_rs/nextclade/src/tree/tree.rs#L83

The options can be set to `None` manually or by serde (in case the field is no present)

Reply:
Yes, it will not appear. This is controlled by the `skip_serializing_if` annotation:

https://github.com/nextstrain/nextclade/blob/650a5eb4a9811d4c8b4bbb9020d7a6c0a5c4a92c/packages_rs/nextclade/src/tree/tree.rs#L81-L83

P.S. the `Option` can be set to `None` manually or by serde (in case the field is no present)